src/views/studio/CreateEditChannelView/CreateEditChannelView.tsx

```diff
@@ -332,6 +332,22 @@ const CreateEditChannelView: React.FC<CreateEditChannelViewProps> = ({ newChanne
           storageProviderUrl
         )
       }
+      if (
+        (data.cover.blob && coverContentId && storageProviderUrl) ||
+        (data.avatar.blob && avatarContentId && storageProviderUrl)
+      ) {
+        displaySnackbar({ title: 'Asset being uploaded', iconType: 'info' })
+      }
+      if (
+        data.cover.blob &&
+        coverContentId &&
+        storageProviderUrl &&
+        data.avatar.blob &&
+        avatarContentId &&
+        storageProviderUrl
+      ) {
+        displaySnackbar({ title: '(2) Assets being uploaded', iconType: 'info' })
+      }
```

Comment:
I think you could refactor this. For example, you could just return if there is no `storageProviderUrl`. You also used `storageProviderUrl` twice 😅  above